### PR TITLE
MG-177 - Add password visibility eye toggle functionality to magistrala login page

### PR DESCRIPTION
**Pull request by wambui-pixel**
# What type of PR is this?

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update
- [ ] Dependency Update

## What does this do?
Adding an eye visibility toggle function to the login page allowing users to toggle on to see the password and toggle off to not see the password.

## Which issue(s) does this PR fix/relate to?

- Related Issue #
- Resolves #177 

## Have you included tests for your changes?

- [x] Yes
- [ ] No, and this is why: 

## Did you document any new/modified functionality?
Yes

- [x] Yes
- [ ] No, and this is why:

### CODE REVIEW

File: ui/web/templates/login.html [hunk 1]
```diff
@@ -51,6 +51,7 @@ <h2>Login</h2>
                         id="password"
                         placeholder="Password"
```
Comment (ianmuchyri) [inline ui/web/templates/login.html:52]: Increase the margin between the input and the eye
image

to do this you can add a bootstrap class `me-2` to the input element. Do this for the envelope in the email-field too and the lock in the password field. They will look sth like 
` <i class="fas fa-solid fa-envelope me-2"></i>`
`  <i class="fas fa-solid fa-lock me-2"></i>`
` <input
                        class="p-3 w-100 me-2"
                        type="password"
                        name="password"
                        id="password"
                        placeholder="Password"
                      /> `

Do this for the other forms as well

File: ui/web/templates/login.html [hunk 2]
```diff
@@ -176,11 +178,26 @@ <h5 class="modal-title" id="forgotPasswordModalLabel">Enter email</h5>
...
                   break;
               }
+
             })
+
             .catch((error) => {
               console.error("error submitting login form: ", error);
             });
         }
+         //code for toggling the visibility of the password
```
Comment (ianmuchyri) [inline ui/web/templates/login.html:188]:
```suggestion
         // Code for toggling the visibility of the password
```

Comment (ianmuchyri) [reply ui/web/templates/login.html:188]: change the other comments as well

**Review by ianmuchyri — CHANGES_REQUESTED** [review]

**Review by ianmuchyri — APPROVED** [review]
LGTM

**Review by Musilah — APPROVED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**